### ISSUE TITLE
Relation editing: Merge "blue" and "yellow" highlighting in the forms

### ISSUE THREAD

**Issue by uncoyote** [opengisch/QgisModelBaker#310]
It is very confusing to have two ways to highlight / select things in the forms. Merge if possible.

**Comment by signedav** [comment]
That's how the attribute table form view works as well. The yellow highlight marks the selection, the blue the index:
![image](https://user-images.githubusercontent.com/28384354/123694512-b4af2c00-d859-11eb-84bd-a1894a8cbbe8.png)

What could be possible improvements? Should the index be marked differently. Like with an indicator instead of a highlight? Any ideas?

**Comment by signedav** [comment]
Indicator idea. Just for the discussion.
![image](https://user-images.githubusercontent.com/28384354/124095102-a891bd00-da59-11eb-8d52-0f36974f6c08.png)

Those checkboxes never containing a checkmark but yellow inside, are maybe not needed then anymore. Yellow should stay the color of the selected features. So we might can mark them yellow (instead of blue) - but I'm not sure if it's too noisy then...

**Comment by domi4484** [comment]
I think if the checkboxes would have a check-mark instead of becoming yellow it would be more clear that this items are "checked" for link/unlink/... The "yellow check" is typical in QGIS but was for example confusing me as a newcomer.
Changing the blue color to something else could be problematic as it comes from the theme and in unfortunate combination with the foreground color could make the text unreadable.

**Comment by andreasneumann** [comment]
I agree with @domi4484 - having a check-mark may imrove the situation to make it clearer that this is about "selection".

**Comment by signedav** [comment]
Maybe once we could create prototypes based on the upcoming linking relation editor.
